ERCS/erc-7770.md

```diff
@@ -0,0 +1,180 @@
+---
+eip: 7770
+title: Fractional Reserve Token
+description: An interface for fractional reserve tokens.
```

Comment:
Your description just restates your title. Use the description to elaborate on the ideas introduced in your title instead. Perhaps explain what a fractional reserve token is?